pulp_rpm/app/viewsets.py

```diff
@@ -369,8 +427,6 @@ class RpmRemoteViewSet(RemoteViewSet):
         ],
         "rpm.rpmremote_creator": [
             "rpm.add_rpmremote",
```

Comment:
i still don't understand why you are keeping the 'add' permission for the owner? it should only have the creator.